src/Workspaces/Remote/Core/ExternalAccess/UnitTesting/Api/UnitTestingAssetStorageWrappper.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal readonly struct UnitTestingAssetStorageWrappper
+    {
+        public readonly static UnitTestingAssetStorageWrappper Instance = new UnitTestingAssetStorageWrappper(AssetStorage.Default);
```

Comment:
💡 If we have to make changes in Roslyn, it will be better if this is a property instead of a field

```suggestion
        public static UnitTestingAssetStorageWrappper Instance { get; } = new UnitTestingAssetStorageWrappper(AssetStorage.Default);
```